### PR TITLE
[MPOM-351] Upgrade plexus-utils 3.4.2

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -966,7 +966,7 @@ under the License.
       <dependency>
         <groupId>org.codehaus.plexus</groupId>
         <artifactId>plexus-utils</artifactId>
-        <version>3.3.0</version>
+        <version>3.4.2</version>
```
Comment (slawekjaranowski) [inline pom.xml:969]: Please also remove 
```
<!-- Commonly shared: last Java7 -->
```
form line 965

**Review by slawekjaranowski — COMMENTED** [review]
Depends on: #83

File: pom.xml (hunk 1 above)
Comment (gnodet) [reply pom.xml:968]: Done

**Review by slawekjaranowski — APPROVED** [review]

**Comment by gnodet** [comment]
Superseded by #87

**Pull request closed without merging**